docs/docs/backup_howto.md

```diff
@@ -0,0 +1,242 @@
+- Faster and more storage-efficient than full backups
+
+**Cons:**
+- Restore requires the initial full backup plus subsequent deltas  
+- Slightly more complex restore process
+
+**Limitations and extensions:**
+- Requires initial full backup  
+- Can be combined with long-term retention
+
+**Best suited usage**
+
+---
+
+### Incremental replication
+**Pros:**
+- only transfer the modified data blocks since last replication 
+- Rapid failover to replicated VM
+
+**Cons:**
+- Requires a secondary host or pool  
+- Not suitable for long-term retention
+
+**Limitations and extensions:**
+- Must keep the chain short ( < 10): Not intended as an archival method
+
+**First steps:**
+1. Configure target storage inside the same pool or another pool
+2. Ensure compatible storage and network settings
+
+---
+
+### Full replication
+**Pros:**
+- Creates a complete VM copy on another storage (same pool or another pool)
+- Ideal for disaster recovery
+
+**Cons:**
+- Longer replication times than incremental  
+- Requires more bandwidth and storage
+
+**Limitations and extensions:**
+- Run less frequently than incremental replication  
+- Suitable for DR plans
+
+**First steps:**
+1. Configure DR target storage (same pool or another pool)
+2. Test failover procedures
+
+---
+
+### Sequence
+**Advantages:**
+- Allows chaining multiple jobs in order  
+- Useful for complex backup and replication workflows
+
+**Cons:**
+- Increases job complexity  
+- Requires careful planning
+
+**Limitations and extensions:**
+- Limited by the capabilities of individual job types
+
+**First steps:**
+1. Identify required job order  
+2. Create and test sequence on non-critical VMs
+
+---
+
+### Mirror backup
+**Pros:**
+- Always keeps the latest backup version
+
+**Limitations and extensions:**
+- Best suited for replication to storage with low bandwidth
+- You can mirror incremental backups or full backups
+
+**First steps:**
+1. Set up a dedicated remote  
+2. Test restore from latest mirror
+
+---
+
+## What settings are available?
+
+### Advanced settings
+- **Smart mode**: Automatically decides whether to run a full or delta backup based on circumstances (such as storage conditions or snapshot issues)
+- **Compression**: Reduces backup size at the cost of CPU usage and backup speed.
+- **Encryption**: Protects backup data at rest. This option is configured at the backup repository level, not per individual backup job.
+- **Concurrency**: Controls how many VMs run in parallel in this backup job
+- **Retention policy**: Fine-tunes how many backups to keep over certain periods of time
+
+---
+
+## Restore options
+
+### VM restore
+- Restore a VM to the same host/pool or another location  
+- Choose full or delta restore, depending on backup type
+
+### File restore
+- Access individual files within a VM backup  
+- Ideal for quick recovery of deleted or corrupted files without a full VM restore
+
+### Limitations
+
+---
```

Comment:
only the simpler partition types are supported: those that can mounted on a debian + NTFS. Especially ReFS and dynamic disks on windows are completly unsupported.  LVM support is partial, and have a lot of edge case for now. 

It is intended to be used to restore a small set of file

for the more complex cases, we provide a script that allow a user to mount the backup chain as a raw disk, and then do the partition discovery and mouting by hand https://github.com/vatesfr/xen-orchestra/tree/master/%40vates/fuse-vhd